src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -416,22 +416,29 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
         })
       }
 
+      if (maybeWithdrawGasEstimateCryptoBaseUnit?.isErr()) {
+        return translate('common.insufficientAmountForGas', { assetSymbol: feeAsset.symbol })
+      }
+
       if (valueCryptoPrecision.isEqualTo(0)) return ''
       return hasValidBalance || 'common.insufficientFunds'
     },
     [
+      opportunityData?.stakedAmountCryptoBaseUnit,
+      maybeOutboundFeeCryptoBaseUnit,
+      maybeWithdrawGasEstimateCryptoBaseUnit,
       amountAvailableCryptoPrecision,
       asset.precision,
       asset.symbol,
-      opportunityData?.stakedAmountCryptoBaseUnit,
-      maybeOutboundFeeCryptoBaseUnit,
       translate,
+      feeAsset.symbol,
     ],
   )
 
   const validateFiatAmount = useCallback(
```

Comment:
Thinking we debounce these for a follow-up - would need to ensure the debounced version uses the latest references on every call however.